AGM_SwitchUnits/clientInit.sqf

```diff
@@ -6,17 +6,16 @@
 0 spawn {
   private ["_side"];
   
-  waitUntil {sleep 0.5; !isNil "AGM_SwitchUnits_EnableSwitchUnits"};
-  if (AGM_SwitchUnits_EnableSwitchUnits == 0) exitWith {};
```

Comment:
@bux578, the above didn't work, because `AGM_SwitchUnits_EnableSwitchUnits` was always already defined before clientInit.sqf was executed. My guess is that the module didn't work if activated after the mission start.